node/src/BaseClient.ts

```diff
@@ -113,16 +114,18 @@ import {
     createZRemRangeByRank,
     createZRemRangeByScore,
     createZScore,
-    createSUnion,
 } from "./Commands";
 import {
     ClosingError,
+    ConfigurationError,
     ConnectionError,
     ExecAbortError,
     RedisError,
     RequestError,
     TimeoutError,
 } from "./Errors";
+import { GlideClientConfiguration } from "./GlideClient";
+import { ClusterClientConfiguration } from "./GlideClusterClient";
```

Comment:
This is not pubsub specific, how did it work before?

Reply:
Also, we need to align ClusteClientConfiguration -> GlideClusterClientConfiguration just as we did with python. (not in this PR)

Reply:
1. we didnt need to use it in g=here until now
2. youre right, added that to one of my tasks

Reply:
Please add the issue link

Reply:
#1974